robottelo/test.py

```diff
@@ -132,13 +134,26 @@ class UITestCase(TestCase):
     def setUpClass(cls):  # noqa
         """Make sure that we only read configuration values once."""
         super(UITestCase, cls).setUpClass()
-        cls.katello_user = settings.server.admin_username
-        cls.katello_passwd = settings.server.admin_password
+
+        cls.katello_default_user = settings.server.admin_username
+        cls.katello_default_password = settings.server.admin_password
```

Comment:
I thought there would be less overhead accessing settings objects only once and then use the cls value for subsequent. But I did not look to see how it is cached